src/static/css/content.less

```diff
@@ -279,6 +276,10 @@ figure {
 
```

Comment:
Need to work on the responsiveness for XS screens:

![screen shot 2017-03-16 at 17 31 25](https://cloud.githubusercontent.com/assets/1044670/24019526/b82639d6-0a6e-11e7-9c99-284f2c7befed.png)